docs/source/data_ingestion_and_preparation/data_preparation_concepts.md

```diff
@@ -25,24 +25,24 @@ docset = context.read\
 
 ### Document
 
-A Document is a generic representation of an unstructured document in a format like PDF, HTML. Though different types of document may have different properties, they all contain the following common fields:
+A Document is a generic representation of an unstructured document in a format like PDF or HTML. Though different types of Documents may have different properties, they all contain the following common fields:
 
-* **doc_id:** A unique identifier for the document. Defaults to a uuid.
+* **doc_id:** A unique identifier for the document. Defaults to a UUID.
 
-* **parent_id:** In Sycamore, certain operations create parent-child relationships between documents. For example, the explode transform promotes elements to be top-level documents, and these documents retain a pointer to the document from which they were created using the parent_id field. For those documents which have no parent, parent_id is None.
+* **parent_id:** In Sycamore, certain operations create parent-child relationships between Documents. For example, the explode transform promotes elements to be top-level documents, and these documents retain a pointer to the document from which they were created using the parent_id field. For those Documents which have no parent, parent_id is None.
 
-* **binary_representation, text_representation:** The raw content of the document in stored in the appropriate format. For example, the content of a PDF document will be stored as the binary_representation, while an HTML page will be stored as text_representation.
+* **binary_representation, text_representation:** The raw content of the Document in stored in the appropriate format. For example, the content of a PDF document will be stored as the binary_representation, while an HTML page will be stored as text_representation.
```

Comment:
This is not correct.  After a ctx.read.binary() you will only have the binary representation for both html and pdf docs. It's the .partition step which makes the text_representation.

```
* **binary_representation:** The raw content of the document. May not be present in elements after partitioning of non-binary inputs such as HTML.
* **text_representation:** The extracted text from the document; this representation is created in the elements after a .partition() step and promoted to top level documents after an .explode() step.
```

Tested both PDF and HTML.

Reply:
Ack.